src/SFA.DAS.EmployerIncentives.Commands/Services/BusinessCentralApi/BusinessCentralFinancePaymentsService.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using SFA.DAS.EmployerIncentives.Abstractions.DTOs.Queries.ApprenticeshipIncentives;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.Exceptions;
+using SFA.DAS.EmployerIncentives.Enums;
+
+namespace SFA.DAS.EmployerIncentives.Commands.Services.BusinessCentralApi
+{
+    public class BusinessCentralFinancePaymentsService : IBusinessCentralFinancePaymentsService
+    {
+        private readonly HttpClient _client;
+        private string _apiVersion;
+        private int _paymentRequestsLimit;
+
+        public BusinessCentralFinancePaymentsService(HttpClient client, int paymentRequestsLimit, string apiVersion)
+        {
+            _client = client;
+            _apiVersion = apiVersion ?? "2020-10-01";
+            _paymentRequestsLimit = paymentRequestsLimit <= 0 ? 1000 : paymentRequestsLimit;
+        }
+
+        public async Task<PaymentsSuccessfullySent> SendPaymentRequestsForLegalEntity(List<PaymentDto> payments)
+        {
+            var paymentsToSend = payments.Take(_paymentRequestsLimit).ToList();
+            var content = CreateJsonContent(paymentsToSend);
```

Comment:
Can you not use PostAsJsonAsync ?

Reply:
Tried it using PostAsJsonAsync and the problem was the camel case format wasn't getting created and teh BC api was rejecting it